adapter-mongodb/src/main/resources/activities/mongodb-basic-uuid.yaml

```diff
@@ -16,7 +15,7 @@ bindings:
 blocks:
   - name: rampup
     tags:
-      phase: rampup
+      block: rampup
```

Comment:
explicit tags for the block names are not needed.

Reply:
I think this is an issue with my local rebase, will get those pulled back in.